src/Scripting/Core/ScriptSourceResolver.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.Scripting
+{
+    using static ParameterValidationHelpers;
+
+    public sealed class ScriptSourceResolver : SourceFileResolver, IEquatable<ScriptSourceResolver>
```

Comment:
This is added for discoverability and consistency with ScriptReferenceResolver. Scripting users should only need to import Microsoft.CodeAnalysis.Scripting(.CSharp);